ngraph/test/runtime/interpreter/int_executable.hpp

```diff
@@ -1229,8 +1229,20 @@ class INTERPRETER_BACKEND_API ngraph::runtime::interpreter::INTExecutable : publ
         case OP_TYPEID::Round:
         {
             size_t element_count = shape_size(node.get_output_shape(0));
-            reference::round<T>(
-                args[0]->get_data_ptr<const T>(), out[0]->get_data_ptr<T>(), element_count);
+            reference::round<T>(args[0]->get_data_ptr<const T>(),
+                                out[0]->get_data_ptr<T>(),
+                                element_count,
+                                op::v5::Round::RoundMode::HALF_TO_EVEN);
+            break;
+        }
+        case OP_TYPEID::Round_v5:
+        {
+            const op::v5::Round* round = static_cast<const op::v5::Round*>(&node);
```

Comment:
If you add evaluate method to the operation, you can use evaluate instead of this code.

Reply:
Ok, this code removed.